### PR TITLE
#BE-2446 Notification Improvement Docs Update

### PR DESCRIPTION
**Pull request by atkncamurcu**

### CODE REVIEW

File: docs/account/email-connection.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ You can also define more than one email address for a module and send notificati
...
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-ReleaseNotesViaEmail.png' />
+
+:::info
+After completing the specified action in Appcircle, you have the option to share the test results via email.
+To enable this feature, ensure you include the [**Test Reports**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report) step in your workflow.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-TestReportsViaEmail.png' />
```
Comment (boztopuz) [inline docs/account/email-connection.md:40]: @csonuryilmaz @atkncamurcu The warning message "Cannot Connect to server" appears in the screen shot background. The screenshot may need to be changed. If it is not necessary, I fixed some typos and added related documentation links to mentioned workflow components.

File: docs/account/slack/slack-notifications.md [hunk 2]
```diff
@@ -54,6 +52,24 @@ Keep scrolling down on Appcircle to see the full list of events.
...
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-ReleaseNotesViaEmail.png' />
+
+:::info
+After completing the specified action in Appcircle, you have the option to share the test results via Slack.
+To enable this feature, ensure you include the [**Test Reports**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report) step in your workflow.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-TestReportsViaEmail.png' />
```
Comment (boztopuz) [inline docs/account/slack/slack-notifications.md:71]: @csonuryilmaz @atkncamurcu The warning message "Cannot Connect to server" appears in the screen shot background. The screenshot may need to be changed. If it is not necessary, I fixed some typos and added related documentation links to mentioned workflow components.

@csonuryilmaz apart of these, ss links are good, template is good. fyi.

**Review by boztopuz — CHANGES_REQUESTED** [review]
@csonuryilmaz I set the PR status as request changes because of the fact that warning message appears in background of screen shots.

File: docs/account/email-connection.md (hunk 1 above)
Comment (csonuryilmaz) [reply docs/account/email-connection.md:40]: @boztopuz We will improve the SS after the PR merge by overwriting the SS with the fixed image.

Thanks for mentioning the case. :pray:

**Review by csonuryilmaz — APPROVED** [review]

**Pull request merged**